docs/ingest/postgres/replication.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Logical Replication
+---
+
+<Note>
+  ParadeDB BYOC (Bring Your Own Cloud), a managed deployment of ParadeDB inside
+  your AWS account, is coming soon. Please fill out the [interest
+  form](https://form.typeform.com/to/jHkLmIzx?typeform-source=paradedb.typeform.com)
+  for access.
+</Note>
+
+In production settings, we recommend deploying ParadeDB as a replica of your primary Postgres. This allows ParadeDB to
+integrate with managed Postgres providers like AWS RDS, Google CloudSQL, Azure Postgres, and others. It also isolates
+search and analytical workloads from your primary Postgres, which can prevent downtime caused by long-running queries.
+
+In this guide, we'll walk through setting up logical replication, also known as streaming replication, from a primary Postgres instance (e.g. AWS RDS) to a ParadeDB deployment.
+
+TODO: Convert this AWS tutorial into our own: https://aws.amazon.com/blogs/database/using-logical-replication-to-replicate-managed-amazon-rds-for-postgresql-and-amazon-aurora-to-self-managed-postgresql/
```

Comment:
This is the only TODO left here. @neilyio I removed your documentation because it did not explain what really matters for users trying to do replication here: ingest from RDS, and specifically configure RDS.

We could perhaps break down logical replication between self-hosted, RDS, etc. and keep your initial docs there, now that It think of it...